docs/develop/create-addin-commands-unified-manifest.md

```diff
@@ -0,0 +1,601 @@
+---
+title: Create add-in commands with the unified Microsoft 365 manifest
+description: Configure the unified Microsoft 365 manifest to define add-in commands for Excel, Outlook, PowerPoint, and Word. Use add-in commands to create UI elements, add buttons or lists, and perform actions.
+ms.date: 04/07/2023
+ms.localizationpriority: medium
+---
+
+# Create add-in commands with the unified Microsoft 365 manifest
+
+Add-in commands provide an easy way to customize the default Office user interface (UI) with specified UI elements that perform actions. For an introduction to add-in commands, see [Add-in commands](../design/add-in-commands.md).
+
+This article describes how to configure the unified manifest to define add-in commands and how to create the code for [function commands](../design/add-in-commands.md#types-of-add-in-commands). 
+
+> [!TIP]
+> Instructions for creating add-in commands with the XML manifest are in [Create add-in commands with the XML manifest](create-addin-commands.md).
+
+> [!NOTE]
+> The unified manifest is currently supported only for Outlook Add-ins. It is in preview and shouldn't be used for production add-ins.
+
+## Starting point and major steps
+
+Both of the tools that create add-in projects with a unified manifest &#8212; the [Office Yeoman generator](yeoman-generator-overview.md) and [Teams Toolkit](teams-toolkit-overview.md)  &#8212;  create projects that initially have one or more add-in commands. So, you'll almost always be working with an add-in project that already has at least one add-in command. One exception is if you are updating an add-in which previously didn't have an add-in command and from which the code and markup for any add-in commands has been removed. 
+
+This article applies to both scenarios, so we use the verb "ensure" for the most part, where "ensure" might mean "create", "add", "rename", or "verify" depending on what your starting point is. For example, "Ensure that there is a 'whatever' property in the 'something' object" means "Verify that there is a 'whatever' ..." when there already is a "whatever" property; but it means "Add a 'whatever' property ..." if there isn't one already.
+
+## Two decisions
+
+Decide which of two types of add-in commands you need: [Task pane or function](../design/add-in-commands.md#types-of-add-in-commands). Decide which kind of UI element you need: button or menu item. Then carry out the steps in the sections and subsections below that correspond to your decisions.
+
+## Add a task pane command
+
+The following subsections explain how to include a [task pane command](../design/add-in-commands.md#types-of-add-in-commands) in an add-in.
+
+### Configure the runtime for the task pane command
+
+1. Open the unified manifest and find the "extensions.runtimes" array.
```

Comment:
I probably missed this discussion, but should "extensions.runtimes" be in codeticks?

Reply:
We're being consistent with Teams docs practices, so we used quotation marks.